package.json

```diff
@@ -58,6 +64,37 @@
     "parcel-bundler": "^1.10.2",
     "parcel-plugin-eslint": "^1.0.4",
     "parcel-plugin-markdown": "mikehenrty/parcel-plugin-markdown#upgrade-asset",
-    "pug": "^2.0.3"
+    "pug": "^2.0.3",
+    "semantic-release": "^15.13.3"
+  },
+  "release": {
+    "ci": false,
+    "repositoryUrl": "git@github.com:Common-Voice/sentence-collector.git",
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@semantic-release/npm",
```

Comment:
mh, I guess we could also just not use that plugin at all and that would be the same?

Reply:
that plugin bumps the package.json version, which is used to figure out the next version afaik.

Reply:
oh, right